### ISSUE TITLE
Open this site in your assigned group?

### ISSUE THREAD

**Issue by daveofthedogs** [Rapptz/power-tabs#55]
Although I selected to permanently open specific tabs in an assigned group, when I open a new browser and try to load the tab, I am stuck on this screen:

![image](https://user-images.githubusercontent.com/783704/91452877-1c1cd980-e84d-11ea-9de0-0f02849d4bed.png)

Regardless or what I check or do, this screen will not load the content, and just sits there.  So, it seems that I can never again load the specific site in my browser when power tabs is loaded :(

Is there some db or cache that can be cleared?

**Comment by texhead** [comment]
Hi Dave, getting the same issue when I open a URL in a different group than what is assigned, but if I am in the group that it is assigned to it opens successfully.

**Comment by gammy** [comment]
Affects me too. No actions on either button; the only solution being to disable the add-on in order to open the link. Debug console shows no messages. Firefox 84.0.2.

**Comment by cppcooper** [comment]
yep this effectively ruins the use of assigning domains for me.. cause sometimes I don't want them to go to the assigned group.. and this makes it an all or nothing situation.

edit: but also, this repo hasn't been updated since 2018. I'm not gonna hold my breath

**Comment by gammy** [comment]
Indeed - do not install this add-on in 2021.
